WordPress/Classes/ViewRelated/Stats/StatsViewController.m

```diff
@@ -85,7 +85,6 @@ - (void)viewDidLoad
 -(void)viewDidDisappear:(BOOL)animated
 {
     [super viewDidDisappear:animated];
```

Comment:
Given that we're now just calling `super`, can we just delete this method altogether/

Reply:
Well, that's been there for like, 5 years. But sure. 😄

Reply:
Git suggests that the following line (removed in this PR) was added on November 9, 2018.
```
[SiteStatsInformation resetSharedInstance];
```

It seems that this removal obviates the need for this method. Even if it were true that this had been in place, I'd hope you agree that we could do better. :)

Reply:
Bah! I misread the method name. Sorry! Yep, yep that's totally my doing.